packages/extension-chakra-storefront/src/components/register/index.jsx

```diff
@@ -41,7 +41,9 @@ const RegisterForm = ({submitForm, clickSignIn = noop, form}) => {
                 <Stack paddingTop={8} gap={8} paddingLeft={4} paddingRight={4}>
                     {form.formState.errors?.global && (
                         <Alert.Root status="error">
-                            <Alert.Indicator />
+                            <Alert.Indicator>
+                                <AlertIcon color="red.500" boxSize={4} />
+                            </Alert.Indicator>
```

Comment:
Brought back the original `AlertIcon`. This is how to properly render a custom icon.. by wrapping it with `<Alert.Indicator>`.